SUPPORT.md

```diff
@@ -0,0 +1,52 @@
+# Support and Compatibility for fabric-chaincode-node
+
+Github is used for code base management, issues should reported in the [FABCN](https://jira.hyperledger.org/projects/FABCN/issues/) component in JIRA.
+
+
+## Summary of Compatibility
+
+Summary of the Fabric versions and the default node runtimes they use to host chaincode and contracts; along side the version of the node modules and which runtimes they are compatible with.
+
+|                       | Fabric 1.4 (node v8) | Fabric v2.0 (node v12) |
+| --------------------- | -------------------- | ---------------------- |
+| node modules **v1.4** |         Yes             |      Yes                  |
+| node modules **v2.0** |         No             |          Yes              |
+
+This is the case by default, however the docker image used to host the chaincode and contracts can be altered by configuring the Fabric Peer.  Set the environment variable `CORE_CHAINCODE_NODE_RUNTIME` on the peer to the name of the docker image.
```

Comment:
Could this have an example docker image name for clarity, I had to read it a couple of times to work out what it wanted?